### DISCUSSION TITLE
Opening files and folders on the terminal

### DISCUSSION THREAD

**Discussion by peterkyle01** [zed-industries/zed#15195] [Feature Requests]
I am on Ubuntu 24.04 LTS using the default terminal.When I try opening folders or files using zed . or zed file_name like the video on [https://zed.dev/linux](https://zed.dev/linux) it doesnt seem to work like the code . or code file_name.Is that feature implemented yet or am I doing it wrongly ?

**Comment by sleeping-at-night** [comment]
I am current run Zed version `0.145.1-1` on arch based. Runing `zeditor .` on my terminal is working correctly. 
Are you sure that you use correct Zed's binary name? Because "Zed" may be the name for program entry, but Zed's binary seem to be `zeditor` on some systems.

**Comment by peterkyle01** [comment]
I looked at the binary name then reinstalled it and it worked using `zed  .` thanks @sleeping-at-night .
